src/schema/v2/__tests__/profile.test.js

```diff
@@ -20,30 +20,30 @@ describe("Profile type", () => {
   const query = `
     {
       profile(id: "the-armory-show") {
-        id
-        is_publically_visible
+        slug
+        isPublicallyVisible
       }
     }
   `
 
-  it("is_publically_visible returns true when profile is published", () => {
+  it("isPublicallyVisible returns true when profile is published", () => {
```

Comment:
very random should it be `isPubliclyVisible`?

Reply:
HA! Yeah, very likely.